test/unit/shim/webframework-shim.test.js

```diff
@@ -618,7 +623,13 @@ test('WebFrameworkShim', function (t) {
         const segment = wrapped(req)
 
         t.ok(segment.attributes)
-        t.same(segment.getAttributes(), {})
+        t.same(segment.getAttributes(), {
+          'code.filepath':
```

Comment:
@bizob2828 same comment as the other

Reply:
i would just update this assertion to make sure there are no request parameters.

```js
const attrs = Object.keys(segment.getAttributes())
const requestParams = /request\.parameters.*/
t.notOk(attrs.some((attr) => requestParameters.test(attr)))
```